lib/prima_opentelemetry_ex.ex

```diff
@@ -25,20 +25,34 @@ defmodule PrimaOpentelemetryEx do
     :ok
   end
 
+  defp is_enabled?(feature) do
```

Comment:
```suggestion
  defp enabled?(feature) do
```
^ this is the usual elixir naming, you either use `is_enabled` (usually for macros) or `enabled?` (usually for functions)

Reply:
Thanks, I didn't know this convention